se/data/templates/content.opf

```diff
@@ -15,8 +15,8 @@
 		<meta property="file-as" refines="#type-designer">League of Moveable Type, The</meta>
 		<meta property="se:url.homepage" refines="#type-designer">https://www.theleagueofmoveabletype.com</meta>
 		<meta property="role" refines="#type-designer" scheme="marc:relators">tyd</meta>
-		<link href="http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa" rel="dcterms:conformsTo"/>
-		<meta property="a11y:certifiedBy">Standard Ebooks</meta>
+		<meta property="dcterms:conformsTo" id="conf">EPUB Accessibility 1.1 - WCAG 2.2 Level AA</meta>
```

Comment:
Can you come up with a more descriptive ID? `conf` can be short for a lot of things, we can use full words separated by dashes if necessary (like `type-designer`)

Reply:
Will do: I think I just copied that out of the spec.